styles/designs/corn/parts/_exercises-components.scss

```diff
@@ -103,7 +103,7 @@ $InjectedExercise__Container: (
       _name: "InjectedExerciseContainer",
       _subselector: ' [data-type="injected-exercise"]',
       _properties: (
-        margin-bottom: (enum('ValueSet:::OPTIONAL'), v-spacing(2)),
+        margin-bottom: (enum('ValueSet:::OPTIONAL'), v-spacing(1)),
```

Comment:
oooh, this is interesting syntax that I wasn't aware of. this is providing the default value of `v-spacing(1)` if nothing else is supplied? i wonder if this may have applications in other places ... 🤔